dev/new-rule/new-rule-test-template.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
```

Comment:
you might have to add these template files to the eslint ignore file to make CI happy 😉